source/includes/_data-dictionary.md.erb

```diff
@@ -19,14 +19,16 @@ Asset characteristics are posted per asset, not per reporting year.
 | construction_year    	| integer                 	| <%= dictionary_description(:construction_year) %>    	|
 | asset_ownership    	  | numeric                 	| <%= dictionary_description(:asset_ownership) %>    	  |
 
-Certifications are posted per asset, not per reporting year.
+Certifications and energy ratings are posted per asset, not per reporting year.
 
 | variable         	| type                    	| description                                      	| unit                 	|
 |------------------	|-------------------------	|--------------------------------------------------	|----------------------	|
 | id 			          | integer                 	| <%= dictionary_description(:id) %>		 	          |                      	|
```

Comment:
this reminds me... 

certifications also have an additional identifier. This is meant to update/overwrite the certification record, in case you have more than one certification with the same brand/scheme, but a different level (gold and platinum for example)

now energy ratings do not have levels, but I do not know if we'll need an `id` to keep track of the energy rating record associated with the energy rating @bartj3? I would say this will not be the case, given there is no need to have two energy ratings with the same `energy_rating_id` associated with the same asset.

Reply:
Thinking about it more, we need an `id` for energy ratings, otherwise how would you delete a energy rating from an asset? https://github.com/GRESB/api-docs/blob/master/source/includes/_certifications.md#remove-certifications

Setting values to `nill` would be a workaround, but let's align it with how certifications work to make it consistent.